services/QuillLMS/app/models/feedback_history.rb

```diff
@@ -241,6 +241,21 @@ def self.list_by_activity_session(activity_id: nil, page: 1, start_date: nil, en
     query
   end
 
+  def self.get_total_count(activity_id: nil, page: 1, start_date: nil, end_date: nil, page_size: DEFAULT_PAGE_SIZE, turk_session_id: nil, filter_type: nil)
```

Comment:
Since `page_size` and `filter_type` are not used in the implementation, we should remove them from the method signature.